source/client/options_impl.cc

```diff
@@ -136,12 +136,19 @@ OptionsImpl::OptionsImpl(int argc, const char* const* argv) {
       "", "max-active-requests",
       fmt::format("Max active requests (default: {}).", max_active_requests_), false, 0, "uint32_t",
       cmd);
-
+  // NOLINTNEXTLINE
   TCLAP::ValueArg<uint32_t> max_requests_per_connection(
       "", "max-requests-per-connection",
       fmt::format("Max requests per connection (default: {}).", max_requests_per_connection_),
       false, 0, "uint32_t", cmd);
 
+  TCLAP::ValueArg<std::string> sequencer_idle_strategy(
+      "", "sequencer-idle-strategy",
+      fmt::format("Choose between using a busy spin/yield loop or have the thread sleep while "
```

Comment:
Probably want to list the legal values here.

Reply:
done